conbench/tests/app/test_runs.py

```diff
@@ -5,8 +5,43 @@
 class TestRunGet(_asserts.GetEnforcer):
     url = "/runs/{}/"
     title = "Run"
-    redirect_on_unknown = False
```

Comment:
This needs to be added back.

The run UI should not redirect to the home page if it doesn't find the run UUID. This is because runs UUIDs are created outside of Conbench by the orchestration layer when a benchmark run kicks off and those run URLs are immediately posted as a GitHub comment. Users should immediately be able to click on those run links, even if benchmark results for that run UUID haven't started coming in yet. Initially, the run page will be an empty datatable, but results will eventually start trickling in.